aws/table_aws_eventbridge_rule.go

```diff
@@ -227,8 +230,10 @@ func getAwsEventBridgeRule(ctx context.Context, d *plugin.QueryData, h *plugin.H
 	}
 
 	// Build the params
+	// If you do not provide value for the EventBusName parameter, By Default DescribeRule API, will only look into the default event bus.
```

Comment:
```suggestion
	// Always provide an event bus name since the default event bus is used if not specified
```

I believe this comment is a bit more descriptive as it includes a bit more info, e.g., we're choosing to always include an event bus name